core/NameHash.h

```diff
@@ -50,7 +50,7 @@ struct UsageHash {
 
 struct FileHash {
     GlobalStateHash definitions;
-    UsageHash usages;
+    UsageHash names;
```

Comment:
WDYT about keeping 2 sets of names separate?
They have semantically different behavior and you rarely want a mix of both, you want either constant names or method names.

Reply:
That's fine with me!